node_type: eagerx_core.basenode/ObservationsNode
default:
  inputs: []
  outputs: [set]
  single_process: true
  launch_locally: true
  color: yellow
inputs: {}
outputs:
  set:
    msg_type: std_msgs.msg/UInt64